src/navigator/DrawerNavigator.tsx

```diff
@@ -251,7 +250,7 @@ export default function DrawerNavigator() {
           {(isCeloEducationComplete && (
             <Drawer.Screen
               name={Screens.ExchangeHomeScreen}
-              component={ExchangeHomeScreen}
+              component={ExchangeHomeScreen as any}
```

Comment:
No way not to cast? If not, I recommend `// @ts-expect-error` instead so we can more easily find it in the future (and remove when unncessary).

Reply:
Fixed in 09194e0af07990b6f94e91594ed218f4f1786b8b!